### PR TITLE
chore: Add mCAKE sdCAKE in stableswap

### PR DESCRIPTION
**Pull request by chef-huan**

### CODE REVIEW

**Review by chefjackson — DISMISSED** [review]

File: subgraphs/exchange-stableswap/subgraph.yaml [hunk 1]
```diff
@@ -30,6 +30,8 @@ dataSources:
           file: ./abis/Factory.json
         - name: Pair
           file: ./abis/Pair.json
+        - name: PairV3
+          file: ./abis/PairV3.json
```
Comment (chefjackson) [inline subgraphs/exchange-stableswap/subgraph.yaml:34]: Need to add to the liquidity event handlers as well

**Review by chefjackson — APPROVED** [review]

File: subgraphs/exchange-stableswap/mappings/utils/index.ts [hunk 2]
```diff
@@ -12,12 +12,14 @@ export let STABLESWAP_FACTORY_ADDRESS_2 = "0x25a55f9f2279A54951133D503490342b50E
 export let PCS_FACTORY_ADDRESS = "0xca143ce32fe78f1f7019d7d551a6402fc5350c73";
 export let BUSD_ADDRESS = "0xe9e7cea3dedca5984780bafc599bd69add087d56";
 export let WBNB_ADDRESS = "0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c";
+export let CAKE_ADDRESS = "0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82";
```
Comment (chefjackson) [inline subgraphs/exchange-stableswap/mappings/utils/index.ts]: should use lowercase

**Review by chefjackson — CHANGES_REQUESTED** [review]
cake address should use lowercase

**Review by chefjackson — APPROVED** [review]

**Review by goalend — COMMENTED** [review]
✌️

**Pull request merged**